src/interactions/panZoomInteraction.ts

```diff
@@ -1,4 +1,8 @@
 namespace Plottable.Interactions {
+
+  export type PanCallback = (e: Event) => void;
+  export type ZoomCallback = (e: Event) => void;
```

Comment:
Not sure these callbacks should take `Event`s -- that's sort of atypical if you compare to the callbacks passed to `Click`, `Drag`, and `Pointer` `Interaction`s. I would expect the final domains of the scales or something like that.